src/modules/Crop/info.json

```diff
@@ -1,5 +1,7 @@
 {
   "name": "Crop",
+  "description": "Crop image to given x, y, w, h",
```

Comment:
these two lines. what do you think?

Reply:
@jywarren The `info.json` is read by the `sequencer.modulesInfo()` method.

for the crop module you could use `sequencer.modulesInfo('crop').description`

I am not sure whether this should be accessible via the `step` parameter. As of now, It can't be accessed by `step`.

Some modules have a `description` property defined as `options.description` like [here in the Invert Module](https://github.com/publiclab/image-sequencer/blob/master/src/modules/Invert/Module.js#L8).

This can be accessed via the `step` parameter because `options` is also accessible via `step` like `step.options`. So the description would be `step.options.description`.

I know this is getting very ugly and needs to be cleaned.

![screen shot - - at](https://user-images.githubusercontent.com/20787612/30001766-4af7a46c-90b5-11e7-8e1a-747e763e1efe.JPG)

Since we added the `info.json` feature, It makes sense to move everything to `info.json`. Yes, you're right, I think this should be worked upon separately.

Reply:
Would it be okay to add the info from `info.js` into the `step`? Like :

```
step = {
    ID: 1,
    imageName: "image1"
    imgElement: <...>,
    ...
    info: {
        /* Info from info.json */
    }
}
```